src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -47,11 +48,19 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TestsSuccessfulSemaphore>$(MSBuildProjectFile).tests.lastsucceeded</TestsSuccessfulSemaphore>
+  </PropertyGroup>
+
   <!-- On command line run unit tests on CoreCLR after the Test target -->
   <Target Name="RunTestsForProject"
           AfterTargets="CheckTestCategories"
+          Inputs="@(PackagesConfig);$(TestRuntimePackageConfig);@(ReferenceCopyLocalPaths);@(Content);@(IntermediateAssembly);@(_DebugSymbolsIntermediatePath);@(AllItemsFullPathWithTargetPath)"
```

Comment:
I would still like these inputs to be pulled out into their own item group as they are shared with the CopyTestToTestDirectory. They aren't going to be used an Inputs to that target but they are used as part of the copy step (i.e. TestCopyLocal item group) and I would perfer we only have one place to keep these sets in sync. For example when you added AllItemsFullPathWithTargetPath you add to add it in both places and if we add others in the future I suspect people will miss one or the other locations.